runelite-client/src/main/java/net/runelite/client/plugins/randomevents/RandomEventPlugin.java

```diff
@@ -179,16 +181,18 @@ private boolean shouldNotify(int id)
 		{
 			case NpcID.SERGEANT_DAMIEN_6743:
 				return config.notifyDemon();
+			case NpcID.DUNCE_6749:
+				return config.notifyDunce();
+			case NpcID.EVIL_BOB:
+			case NpcID.EVIL_BOB_6754:
+				return config.notifyBob();
```

Comment:
I don't see any benefit in reordering these.

Reply:
removed